lib/Autocomplete.js

```diff
@@ -332,8 +338,8 @@ let Autocomplete = React.createClass({
           aria-autocomplete="both"
           autoComplete="off"
           ref="input"
-          onFocus={this.handleInputFocus}
-          onBlur={this.handleInputBlur}
+          onFocus={(event) => this.handleInputFocus(event)}
+          onBlur={(event) => this.handleInputBlur(event)}
```

Comment:
This is not necessary, you just create an additional function and extra indirection for nothing. onFocus={callback} will call callback with the event already, much like it calls this inline function with the event.